applications/produce_array_config.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python3
+
+'''
+    Summary
+    -------
+    This application is a simple example of how to produce sim_telarray config \
+    files for a given array.
+
+    All the input required has to be given as a yaml file by the command \
+    line argument array_config.
+
+    The required entries in the array_config file are:
+
+    'site': South or North
+
+    'layoutName': name of a valid layout array.
+
+    'modelVersion': name of a valid model version.
+
+    'default': telescope model names to be assigned to each telescope size by default. \
+    It must contain entries for 'LST' and 'MST' (and 'SST' in case of South site).
```

Comment:
Not sure if I understand correctly: 'it must contain entries for 'LST'' --> does this mean that arrays without LSTs are not possible to define?

Reply:
The 'default' model is required.

The other option is checking whether all the telescopes were set individually and allow for that. I don't think this will be very useful in practice, but we could do that.

Reply:
Fixed in https://github.com/gammasim/gammasim-tools/pull/110/commits/1bc0416bc8ec6114b8a95222dd14badbdf9a8ddc